V2 Operational Excellence/specs/Tech_Plan__Operational_Excellence_Implementation.md

```
@@ -0,0 +1,1532 @@
+
+  private
+
+  def calculate_current_eta
+    running_attacks = attacks.with_state(:running)
+    return nil if running_attacks.empty?
+
+    # Return the maximum ETA of all running attacks
+    etas = running_attacks.map(&:estimated_finish_time).compact
+    etas.max
+  end
+
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)